.github/workflows/workflow.yaml

```diff
@@ -33,25 +33,13 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Set up Go 1.16
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.16
-
-      - name: Install golangci-lint
-        run: |
-          mkdir -p $GITHUB_WORKSPACE/bin
-          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
-          curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $GITHUB_WORKSPACE/bin v${GOLANGCILINT_VERSION}
-        env:
-          GOLANGCILINT_VERSION: 1.40.1
-
-      - name: Make lint
-        run: |
-          make lint
+      # source: https://github.com/golangci/golangci-lint-action
```

Comment:
nit: comment needs to moved 1 line below

Reply:
Done.